test/cpuinfo_x86_test.cc

```diff
@@ -1172,6 +1172,21 @@ TEST_F(CpuidX86Test, INTEL_CML_H) {
   EXPECT_EQ(GetX86Microarchitecture(&info), X86Microarchitecture::INTEL_CML);
 }
 
+// https://github.com/InstLatx64/InstLatx64/blob/master/GenuineIntel/GenuineIntel00A0660_CometLake_CPUID1.txt
+TEST_F(CpuidX86Test, INTEL_CML_U2) {
+  cpu().SetLeaves({
+      {{0x00000000, 0}, Leaf{0x00000016, 0x756E6547, 0x6C65746E, 0x49656E69}},
+      {{0x00000001, 0}, Leaf{0x000A0660, 0x00100800, 0x7FFAFBBF, 0xBFEBFBFF}},
+  });
+  const auto info = GetX86Info();
+
+  EXPECT_STREQ(info.vendor, "GenuineIntel");
```

Comment:
I created PR to replace values to constants https://github.com/google/cpu_features/pull/265, could you change to `CPU_FEATURES_VENDOR_GENUINE_INTEL` for consistency, please?